collector/mongodb_collector.go

```diff
@@ -126,3 +132,28 @@ func (exporter *MongodbCollector) collectDatabaseStatus(session *mgo.Session, ch
 		}
 	}
 }
+
+func (exporter *MongodbCollector) collectCollectionStatus(session *mgo.Session, ch chan<- prometheus.Metric) {
+	database_names, err := session.DatabaseNames()
+	if err != nil {
+		glog.Error("failed to get database names")
+		return
+	}
+	for _, db := range database_names {
+		if db != "admin" && db != "test" {
```

Comment:
please reduce the number of nested blocks in this section of code. for example:

```go
if db == "admin" || db == "test" {
        continue
}
```
you can also move some code to helper functions/methods to accomplish so. The less blocks the better